addons/stock/i18n/stock.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 16.0beta\n"
+"Project-Id-Version: Odoo Server 16.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-10-10 10:22+0000\n"
-"PO-Revision-Date: 2022-10-10 10:22+0000\n"
+"POT-Creation-Date: 2022-10-24 11:28+0000\n"
+"PO-Revision-Date: 2022-10-24 11:28+0000\n"
```

Comment:
No need to edit that. Remove it and I will push the PR

Reply:
hi @amoyaux updated the po file